support-models/src/main/scala/com/gu/support/workers/ProductTypeRatePlans.scala

```diff
@@ -39,6 +39,14 @@ object ProductTypeRatePlans {
       .getOrElse(environment, Nil)
       .find(productRatePlan => productRatePlan.billingPeriod == product.billingPeriod)
 
+  def guardianLightRatePlan(
+      product: GuardianLight,
+      environment: TouchPointEnvironment,
+  ): Option[ProductRatePlan[catalog.GuardianLight.type]] =
+    catalog.GuardianLight.ratePlans
+      .getOrElse(environment, Nil)
+      .find(productRatePlan => productRatePlan.billingPeriod == product.billingPeriod)
```

Comment:
Revisit this now we've removed `billingPeriod` from the product. Maybe just grab the first (assuming there will only be one).